packages/openapi-framework/test/sample-projects/operations-dir-with-valid-methid-doc-after-dependency-injection/spec.ts

```diff
@@ -45,6 +51,19 @@ describe(path.basename(__dirname), () => {
             },
           },
         });
+        expect(apiDoc.paths['/foo-two']).to.eql({
```

Comment:
i think you should add a new controller that returns the operation doc, then verify that the 2 paths using the same controller return different docs according to the path they're mounted for.  ya know?

Reply:
I feel like this test _mostly_ captures that already since both operations api docs get asserted.

I understand how to implement what you're asking for in express-openapi, but not sure how I'd do it here. I'll read around the codebase some more, but pointers would be appreciated.

Reply:
Implemented this in the express package for now.